requirements/requirements.in

```diff
@@ -2,12 +2,13 @@ attrs~=18.1
 iso8601==0.1.12
 jsonobject==0.9.8
 jsonobject-couchdbkit~=0.9
-celery==3.1.25
+celery~=4.2
```

Comment:
FYI that they just released a release candidate for 4.3 so this might try to upgrade in next week or two.